src/public/index.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  CSVUploader,
+  CSVEntityPicker,
+  CSVEntityName,
+  CSVEntityProperties,
+  CSVEdgeMapping,
+  CSVImportFeedback,
+} from "./cards";
+import { EntitiesTypes } from "./models";
+import * as utils from "./utils";
+
+function main() {
+  let entityType: EntitiesTypes;
+
+  /************** Initialize plugin  ************/
+
+  const uploader = new CSVUploader();
+  uploader.init();
+
+  const entityPicker = new CSVEntityPicker();
+  entityPicker.init();
+
+  const entityName = new CSVEntityName();
+  entityName.init();
+
+  const entityProperties = new CSVEntityProperties();
+  entityProperties.init();
+
+  const edgeMapping = new CSVEdgeMapping();
+  edgeMapping.init();
+
+  const importFeedback = new CSVImportFeedback();
+  importFeedback.init();
+
+  /************** Set event handlers ************/
+
+  // cancel button (go to first page and reset state)
+  const cancelButtons = document.getElementsByClassName(
```

Comment:
maybe need to add a warning comment next to the cancelButton class  to avoid using it in another button that  it is not supposed to reset the plugin

Reply:
added a warning and changed the class to resetPluginAction to make it clearer even. it is not used in css so should not be added somewhere for design purpose which is a good thing also. Risk mitigated